rollup/internal/controller/permissionless_batches/minimal_recovery.go

```diff
@@ -0,0 +1,362 @@
+package permissionless_batches
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/scroll-tech/da-codec/encoding"
+	"github.com/scroll-tech/go-ethereum/core"
+	"github.com/scroll-tech/go-ethereum/ethclient"
+	"github.com/scroll-tech/go-ethereum/log"
+	"github.com/scroll-tech/go-ethereum/rollup/l1"
+	"gorm.io/gorm"
+
+	"scroll-tech/common/types"
+	"scroll-tech/database/migrate"
+	"scroll-tech/rollup/internal/config"
+	"scroll-tech/rollup/internal/controller/watcher"
+	"scroll-tech/rollup/internal/orm"
+)
+
+const (
+	defaultRestoredChunkIndex  uint64 = 1337
+	defaultRestoredBundleIndex uint64 = 1
```

Comment:
what would these parameters be in final version? some configurable params?

Reply:
Added a comment for it in the code as well. These are arbitrary numbers. We don't know the correct number for chunk nor bundle. So we can use any random number as a dummy index.